jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3TreeTable.kt

```diff
@@ -158,8 +158,10 @@ class S3TreeTable(
     }
 
     init {
-        // Associate the drop target listener with this instance which will allow uploading by drag and drop
-        DropTarget(this, dropTargetListener)
+        if (!ApplicationManager.getApplication().isHeadlessEnvironment) {
```

Comment:
Why was this added? add a comment on why?

Reply:
Sure, DND can't be started when we are running headless (aka unit tests)

Reply:
using isUnitTestMode probably clearer